### ISSUE TITLE
Crash on forge-34.1.19 1.16.3 startup (Might have to do with Silent Gems)

### ISSUE THREAD

**Issue by P3rf3ctXZer0** [SilentChaos512/Silent-Gear#238]
## Versions
1.16.3

- **Silent Gear**: SilentGear-1.16.3-2.3.1+166
- **Silent's Gems**: SilentGems-1.16.3-3.7.0+103
- **Silent Lib**: SilentLib-1.16.3-4.8.1+61
- **Forge**: forge-34.1.19
- **Modpack**: N/A 
- **Optifine Installed**: No 

## Expected Behavior
No crashing

## Actual Behavior
Crash on startup

## Links/Images
[Evolved 1.16.X-2.0.0-Crash01.zip](https://github.com/SilentChaos512/Silent-Gear/files/5367412/Evolved.1.16.X-2.0.0-Crash01.zip)
[crash-2020-10-12_15.48.11-fml.txt](https://github.com/SilentChaos512/Silent-Gear/files/5367426/crash-2020-10-12_15.48.11-fml.txt)
[latest.log](https://github.com/SilentChaos512/Silent-Gear/files/5367427/latest.log)

## Steps to Reproduce the Problem
I don't know, so I included the not publically available pack.

**Comment by SilentChaos512** [comment]
Yeah, I need to update Gems to be compatible with the latest Silent Gear. You can downgrade to 2.3.0 for now, 2.3.1 is just a few API changes.

**Comment by P3rf3ctXZer0** [comment]
Okay sounds good glad I could help you as always though :P
